cmd-queue.c

```diff
@@ -230,7 +231,8 @@ cmdq_get_command(struct cmd_list *cmdlist, struct cmd_find_state *current,
 
 		log_debug("%s: %s group %u", __func__, item->name, item->group);
 
-		shared->references++;
+		if (shared)
```

Comment:
Is this conditional still necessary? Or should the `if` at the start of this loop have `!shared ||` added to its condition?